components/Turnstile.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <VueTurnstile
+    v-model="token"
+    :reset-interval="resetInterval"
+    :size="size"
+    :theme="theme"
+    :action="action"
+    :appearance="appearance"
+    :render-on-mount="renderOnMount"
+    :site-key="siteKey" />
+</template>
+
+<script setup>
```

Comment:
We've avoided using `<script setup>` because it's some extra macro magic that does some things that can be hard to understand or keep track of / control. I'm not inclined to hold this up for it, so I'll file an issue, but we should probably continue to avoid it because there's just a bit too much magic going on -- that takes us further away from easily understandable javascript and instead hides things behind a compiler / transpiler that then needs to be understood to sort out bugs.